src/App.js

```diff
@@ -1,8 +1,13 @@
 import { Component } from 'react';
-import Calculator from './Calculator';
+import Calculator from './components/Calculator';
 
 // eslint-disable-next-line react/prefer-stateless-function
```

Comment:
- [ ] You have **disabled linter check** on your code. Please enable linter check on your code to ensure that your code is robust code and clean.